src/common/options/osd.yaml.in

```diff
@@ -520,6 +520,64 @@ options:
     stats (inc. scrub/block duration) every this many seconds.
   default: 120
   with_legacy: false
+- name: osd_scrub_retry_delay
+  type: int
+  level: advanced
+  desc: Period (in seconds) before retrying a specific PG following a scrub failure
+  long_desc: Minimum delay after a failed attempt to scrub a PG. See the
+    'see also' for the configuration options for some specific delay reasons
+  default: 30
+  min: 1
+  see_also:
+  - osd_scrub_retry_pg_state
+  - osd_scrub_retry_after_noscrub
+  with_legacy: false
+- name: osd_scrub_retry_after_noscrub
+  type: int
+  level: advanced
+  desc: Period (in seconds) before retrying to scrub a PG at a specific level
+    after detecting a no-scrub or no-deep-scrub flag
+  long_desc: Minimum delay after a failed attempt to scrub a PG at a level
+    (shallow or deep) that is disabled by cluster or pool no-scrub or no-deep-scrub
```

Comment:
I might take out the mentions of level here

Reply:
@anthonyeleven: We now have two scheduled 'targets' per each PG: one for its next shallow scrub, and one for
the next deep one.  I am trying to convey the fact that a specific one of these targets is to be postponed.

Reply:
Wouldn't that require two options?  osd_shallow_scrub_retry_after_noscrub and osd_deep_scrub_retry_after_noscrub?

Reply:
The delay is one - but it is applied to the relevant target - the relevant level. The one that was scheduled to execute - and aborted because of the operator setting the flag.
I have pushed a new version, with the rest of the fixes you have suggested. And in the description of
the default conf - I've expanded a bit about the two levels.